src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -141,6 +141,14 @@ export class NetCoreTaskHelper implements TaskHelper {
         runOptions.containerName = runOptions.containerName || getDefaultContainerName(context.folder.name);
         runOptions.os = runOptions.os || 'Linux';
         runOptions.image = inferImageName(runDefinition as DockerRunTaskDefinition, context, context.folder.name, 'dev');
+        if (runDefinition.netCore?.enableDebugging) {
+            if (runOptions.os === 'Linux') {
+                runOptions.entrypoint = runOptions.entrypoint || '/bin/sh';
+            }
+            else if (runOptions.os === 'Windows') {
+                runOptions.entrypoint = runOptions.entrypoint || 'cmd.exe';
+            }

```

Comment:
You can simplify by using assign if null and a ternary operator, since the only two possible options are Windows or Linux.

```suggestion
            runOptions.entrypoint ??= runOptions.os === 'Windows' ? 'cmd.exe' : '/bin/sh';
```

Reply:
makes sense!